SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# Author: Rodrigo Sobral 2024
+# Copyright CERN
+# This script allows to create a venv virtual environment to use in notebooks and terminals. The environment contains the packages from a provided requirements file.
+
+# By default, we use the Python installed in the image
+ACCPY_BASE=$ACCPY_PATH/base
+ACCPY_ALL_VERSIONS_STR="?"
```

Comment:
No need to initialize to "?", if it's empty nothing will be printed.

Reply:
I know, but I thought it could get weird for the users who see "options:" empty, with nothing after it. Even tho this help page won't be print that much (hopefully), do you think I should remove it just to keep the code simpler?

Reply:
Isn't it weirder to get "options: ?" :D I would just remove it.

Reply:
nahh, the user would be like "i dont know what versions to use"
and this way we're basically saying "neither do we"